Src/FluentAssertions/Equivalency/Selection/ExcludeMemberByPathSelectionRule.cs

```diff
@@ -23,6 +23,12 @@ public ExcludeMemberByPathSelectionRule(MemberPath pathToExclude)
                 memberToExclude.IsSameAs(new MemberPath(member, parentPath)));
         }
 
+        public void ExtendPath(MemberPath nextPath)
+        {
+            memberToExclude = memberToExclude.Extend(nextPath, "[*]");
```

Comment:
🤔 To align with the new `WithMapping` options, I think we should use `[]` for that.

Reply:
Please see this comment https://github.com/fluentassertions/fluentassertions/pull/1782#discussion_r798841929

Reply:
Done 🙂